inc/features/blocks/namespace.php

```diff
@@ -187,7 +198,7 @@ function sanitize_id( $param ) : string {
  */
 function check_views_permission() : bool {
 	$type = get_post_type_object( Audiences\POST_TYPE );
-	return current_user_can( $type->cap->read );
+	return true; // current_user_can( $type->cap->read );
```

Comment:
Is this still in progress/testing? If you're explicitly returning true here, the `$type` variable isn't being used.

Reply:
Sorted, just forgot this one 👍